app/views/course/assessment/submission/submissions/_buttons.html.slim

```diff
@@ -0,0 +1,23 @@
+- submission = f.object
+div.submission-buttons
+  - if defined?(save) && save
```

Comment:
Not sure if we can use this - `if local_assigns[:save]` instead. This `local_assigns` was removed in the API docs, but added back in from rails 5.0 onwards. 

Reference: http://nithinbekal.com/posts/rails-optional-locals/